policyengine_us/parameters/gov/states/dc/tax/income/subtractions/disability_income_exclusion/amount.yaml

```
@@ -0,0 +1,18 @@
+description: DC reduces the excludable disability income by this amount.
+metadata:
+  unit: currency-USD
+  period: year
+  label: DC disability income reduction amount 
+  reference:
+    - title: Code of the District of Columbia § 47–1803.02. (a)(2)(M)
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1803.02
```

But this source doesn't say $15k. Is it defined elsewhere, or does "certain" mean they're delegating to the OTR? If the latter add a comment.

Same for the other parameter as well.
```suggestion
      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1803.02#(a)(2)(M)
```

---

The link below this line has the amount $15k in it, it's the IRC in 1983

---

I see, let's move these parameters to the irs folder then. Ideally we'd backdate the logic for disability payment exclusions but for now we can have them as isolated parameters. Then link the DC Code in the variable instead, since it won't have any parameters attached.

---

How can I link the DC code in the variable? Should I create a `disability_exlucsions.py` under the IRS folder and then put `adds[dc_disability_exclusion]` in it?

---

Example:

https://github.com/PolicyEngine/policyengine-us/blob/5f920f0bee13d0633d1c202b6bf7ac4a2a805fa8/policyengine_us/variables/gov/ssa/ssi/ssi.py#L11